css-three-column-layout/styles.css

```diff
@@ -0,0 +1,71 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  background-color: #1996da;
+  font-family: 'Lato', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  color: #fff;
+}
+
+.title {
+  padding: 2rem;
+  padding-bottom: 1rem;
+  font-size: 1.5rem;
+  letter-spacing: 1px;
+  text-align: center;
+}
+
+section.all {
+  display: flex;
+  justify-content: space-between;
+  width: 60%;
+  margin: 0 auto;
+}
+
+.first,
+.second,
+.third {
```

Comment:
Rather than three separate classes, combine this into one class named something like `.column-third`